api/controllers/adminController.js

```diff
@@ -1,8 +1,7 @@
 'use strict'
-
+const sha3 = require('js-sha3').sha3_224;
```

Comment:
can we not have written as this? the require just just include the require itself and then another variable to have the specific method (or whatever that is) that you want.

Reply:
didn't understand what you wanted

Reply:
const js_sha3 = require('js-sha3');

... (other code from imports)

const sha3 = js_sha3.sha3_224;